example/semaphore-prepare

```diff
@@ -1,2 +0,0 @@
-docker pull fedora:25
-
```

Comment:
Why you removed the example directory? README still mentions example directory.

Reply:
Good point, thanks! I removed it from README, too. As to the "why", it was rather bogus -- most semaphore scripts weren't ever been used, and this wasn't an example for something cockpituous-y at all.